docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+Example:
 
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+* `liste today 17:00 to tomorrow 18:00`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+**View overdue task** : 
 
-Examples:
+Format: `listo`
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+**View tasks and events in the expired list**: 
 
-### 2.6. Deleting a person : `delete`
+Format: `listc`
 
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+NOTE:
+> note that tasks and events have different fields, and therefore the parameter SORT_FIELD may vary between events and tasks.
 
-> Deletes the person at the specified `INDEX`. <br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+### 3.4 Updating a field of a task/event in the active list : `edit`
 
-Examples:
+> User can edit the fields of the tasks and events in the active list by using their indexes specified in the most recent list. The steps are therefore as follows:
+
+1. Use `list` command
+2. Using the index, call `edit INDEX etc.`
+
+**Updating the fields**:
+
+Format: `edit INDEX FIELD_NAME NEW_VALUE`
+
+Examples: 
+
+`edit 1 priority 3`, assuming that the item with index 1 is a task.
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+### 3.5 Mark as `done` or dimiss tasks and events: `done` and `dis`
+> Just like the `edit` command, the user makes use of the index provided by the `list`-type commands. Either way, the task/event gets stored in the expired list.
 
-### 2.7. Select a person : `select`
+**Marking a task or event as done**:
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+Format: `done INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+**Dismissing a task or event, without actually completing**:
+
+Format: `dis INDEX [\\REASON]`
+
+Example:
+
+`dis 2 \\dropped the module so no longer need to complete this assignment`
+
+### 3.6. Finding event or task by keyword: `find`
+
+> Finds tasks or events whose field(s) contain any of the given keywords.<br>
```

Comment:
does it mean search is done on any/all fields in the task? if so, this is good!! if not need to specify.